### PR TITLE
Edit VS Tools for Tizen content

### LINKED ISSUE #46: [Request] Reviewing VS Tools for Tizen
안녕하세요 김소영 차장님.

 VS Tools for Tizen 1.0이 10월 말 릴리즈에 포함됩니다.

그래서 문서를 1.0 에 맞게 담당자께서 작성해 주셨는데요, 사내 GitHub을 사용하고 계셔서 Markdown 로 작성하셨습니다.

그래서 저희한테 영문 검수가 필요한 파일을 주셨는데, tizen-docs github에 넣었습니다. MD 파일을 그대로 영문 검수해주시면 이 파일을 VS Tools 사내 GitHub에도 복사하고 tizen-docs 에 나중에 그대로 사용하고, TD에는 MD 파일을 HTML로 editing 작업을 하도록 하겠습니다.

아래 문서에 대해 영문 검수만 부탁드립니다. (tizen-docs를 공식적으로 사용하는 시점에 문서 path와 directory path는 맞추도록 하겠습니다.)

작업하셔야할 branch는 vs-tools-tizen 입니다.

제목 | developer tizen 문서 위치 | 신규 문서 위치
-- | -- | --
Installing Visual Studio Tools for Tizen | https://developer.tizen.org/development/tizen-.net-preview/getting-started/installing-visual-studio-tools-tizen | tizen-docs/docs/vs-tools-tizen/how-to-install.md
Project Wizard | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/project-wizard | tizen-docs/docs/vs-tools-tizen/tools/project-wizard.md
Certificate Manager | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/certificate-manager | tizen-docs/docs/vs-tools-tizen/tools/certificate-manager.md
Tizen Manifest Editor | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/tizen-manifest-editor | tizen-docs/docs/vs-tools-tizen/tools/manifest-editor.md
Emulator Manager | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/emulator-manager | tizen-docs/docs/vs-tools-tizen/tools/emulator-manager.md
Emulator Control Panel | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/emulator-control-panel | 삭제 (VS Tools 쪽 추가 피쳐 없음)
API and Privilege Checker | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/api-and-privilege-checker | tizen-docs/docs/vs-tools-tizen/tools/api-and-privilege-checker.md
Device Manager | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/device-manager | tizen-docs/docs/vs-tools-tizen/tools/device-manager.md
Log Viewer | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/log-viewer | tizen-docs/docs/vs-tools-tizen/tools/log-viewer.md
Profiler | 신규 피쳐 | tizen-docs/docs/vs-tools-tizen/tools/profiler-user-manual.md

### PR DESCRIPTION
**Pull request by l10nbridge**
Performed basic editing, fixing language and structure.
Improved usability and readability.

Fixes #46.

### CODE REVIEW

File: docs/vs-tools-tizen/tools/emulator-manager.md [hunk 1]
```diff
@@ -1,148 +1,141 @@
...
+4. Change properties as needed, and click **Finish**.
 
-To run the application faster, switch on CPU VT and GPU. If CPU VT is disabled, check Increasing the Application Execution Speed for more information. If GPU is disabled, install the latest vendor-provided graphic driver.
+The emulator instance appears in the Emulator Manager. To view the emulator instance details, right-click the emulator and select **Detail**.
 
-Mobile emulator supports HD(720x1280) or WVGA(480x800) resolutions.
-```
+> **Note**
+>
+> To run the application faster, switch on CPU VT and GPU. If CPU VT is disabled, check Increasing the Application Execution Speed for more information. If GPU is disabled, install the latest vendor-provided graphic driver.
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/tools/emulator-manager.md]: “If CPU VT is disabled, check Increasing the Application Execution Speed for more information.” is missing a link. Where should the link lead, or should this sentence be removed?

File: docs/vs-tools-tizen/how-to-install.md [hunk 2]
```diff
@@ -1,129 +1,138 @@
...
+
+- To check whether Tizen Project properties have been copied correctly:
+  - Make sure that the `Tizen.NET.ProjectType.props` and `Tizen.NET.ProjectType.targets` files are located in the `C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\Tizen\VisualStudio` folder.
+
+     ![Project Property](media/cps-project-property.png)
+
+- To check the Nuget server for build tasks:
+  - Check the package server in **Tools &gt; Option &gt; NuGet Package Manager &gt; Package Sources &gt; https://tizen.myget.org/F/dotnet/api/v3/index.json**.
+
+- If you cannot remove the Tizen extension:
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/how-to-install.md]: The step list makes no sense. What exactly needs to be done in the Visual Studio directory after organizing the files based on the timestamp? What needs to be checked in the folder?

File: docs/vs-tools-tizen/tools/profiler-user-manual.md [hunk 3]
```diff
@@ -1,137 +1,154 @@
...
+        Trace memory allocation and source lines, but not CPU.
+      - **Complex Profiling**  
+        Trace CPU, source lines, and memory allocation.
+    - **Sampling Options**  
+    You can set the following sampling options:
+      - **Sampling Interval**  
+        You can set the sampling time interval, in milliseconds. The default is 10 ms.
+      - **High Granularity**  
+        You can switch on high-granularity sampling. It enables better precision and sample regularity, at the cost of efficiency.
+    - **Trace Options**
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/tools/profiler-user-manual.md]: The "Trace Garbage Collection" option visible in the screenshot is missing a description in this section.

File: docs/vs-tools-tizen/tools/profiler-user-manual.md [hunk 4]
```diff
@@ -1,137 +1,154 @@
...
+3. To start the profiling session, click **OK**.  
+    The application is installed and launches on the emulator.  
+    The **Output** panel displays the installation log, including any errors.  
+    ![Output panel](media/profiler-output-window.png)
+
+    The **Profiling Progress** panel shows live CPU and memory usage information.  
+    ![Profiling progress](media/profiler-progress.png)
+
+4. To pause and resume profiling, in the **Profiling Progress** panel, click **Pause** and **Resume**.  
+    For the time interval when profiling is paused, the profiling chart shows a light-blue background.
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/tools/profiler-user-manual.md]: What exactly does this mean? The light blue background is also visible after resuming.

File: docs/vs-tools-tizen/tools/device-manager.md [hunk 5]
```diff
@@ -89,69 +81,55 @@ namespace CrossTemplate1.Tizen.Mobile
...
+- **Message**: log message
 
-![](../image/dm_log_add_tab.png)
+### Adding and Removing Log Tabs
 
-![](../image/dm_log_added_tab.png)
+To add a log tab:
 
-Removing a log tab:
+1. Click the **Add LogTab** icon.
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/tools/device-manager.md]: Where is this icon? Does it actually refer to the “+” sign on the empty tab?

File: docs/vs-tools-tizen/tools/device-manager.md [hunk 6]
```diff
@@ -89,69 +81,55 @@ namespace CrossTemplate1.Tizen.Mobile
...
 
-### Understanding and Filtering Logs ###
+![Added log tab](media/dm-log-added-tab.png)
 
-The log table consists of the following fields:
+To remove a log tab:
 
-* Time: log time
+1. Click the log tab to be removed.
+2. Click the **Remove LogTab** icon.
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/tools/device-manager.md]: Where is this icon? Does it actually refer to the “x” sign on the existing tab?

File: docs/vs-tools-tizen/tools/device-manager.md [hunk 7]
```diff
@@ -89,69 +81,55 @@ namespace CrossTemplate1.Tizen.Mobile
...
+- Log level filtering option  
+  In the **Log View**, you can filter the log messages. Click the arrow in the **Level** field, select the messages types you want to see, and click **OK**.
 
-* Keyword filtering option
-    * You can filter the log messages by selecting one of the PID, Tag, and Message options, or entering keywords at the search box which is in the bottom of the log view. The keywords must be separated by a space and a comma.
+  ![Log level filter](media/dm-log-level-filter.png)
 
-    ![Log Level Filter](../image/dm_log_filter_option.png)
+- Keyword filtering option  
+  You can filter the log messages by clicking the arrow in the **Pid** and **Tag** fields, selecting the wanted values, and clicking **OK**. To filter based on the **Message** field content, enter keywords at the search box at the top of the **Log View**. The keywords must be separated by a space and a comma.
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/tools/device-manager.md]: “space and a comma”: Should this be “space or comma”?

File: docs/vs-tools-tizen/tools/log-viewer.md [hunk 8]
```diff
@@ -1,57 +1,42 @@
-## Log Viewer ##
+# Log Viewer
 
-Log Viewer enables that you can check your log messages in Visual Studio. This tool shows the messages same as log viewer in device manager. However, this tool can be imported in Visual Studio.
+The Log Viewer enables you to check your log messages in Visual Studio. It shows the same messages as the **Log View** in the [Device Manager](device-manager.md). However, the Log Viewer can be imported into Visual Studio.
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/tools/log-viewer.md]: “the Log Viewer can be imported into Visual Studio”: What does this mean?

File: docs/vs-tools-tizen/packaging/how-to-create-tpk.md [hunk 9]
```diff
@@ -1,72 +1,104 @@
...
 ## Prerequisites
 
-Add feed to Nuget.config (https://tizen.myget.org/F/dotnet/api/v3/index.json) (develop channel) to restore Tizen.NET.Sdk develop version.
+Make sure the following prerequisites are fulfilled before building a project:
 
-1. Visual Studio 2017
+1. Install the needed software based on whether you want to use Visual Studio or .NET CLI:
 
-![Nuget Tizen.Net.Sdk](../image/nuget-tizen.net.sdk.png)
+   - [Install Visual Studio Tools for Tizen](../how-to-install.md) (cps version).
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/packaging/how-to-create-tpk.md]: What is a "cps version"?

File: docs/vs-tools-tizen/packaging/how-to-customize-tpk.md [hunk 10]
```diff
@@ -1,418 +1,434 @@
-# How to customize TPK package
+# Customizing TPK Packages
+
+You can customize TPK packages in various ways.
+
+For more information, see:
```
Comment (l10nbridge) [inline docs/vs-tools-tizen/packaging/how-to-customize-tpk.md]: These links do not work.

File: docs/vs-tools-tizen/how-to-install.md [hunk 11]
```diff
@@ -1,129 +1,138 @@
...
-### Java Development Kit (JDK) Requirements ###
-You must install a Java Development Kit (JDK) 8 or higher to use Tizen Baseline SDK. Do not install OpenJDK.
+## Prerequisites
+To work with Visual Studio Tools for Tizen, your computer must have:
 
-Download the JDK from the [official Oracle Web site](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Select the appropriate platform for your hardware architecture and Windows® version. Then, run the downloaded execution file and follow the displayed instructions.
+- At least 1.5 GB of available disk space
+- Visual Studio 2017  
+Visual Studio Tools for Tizen works with all Visual Studio variations, including Community. Installing or re-installing Visual Studio with .NET desktop development and .NET Core cross-platform development toolsets is recommended.
+- .NET Core 2.0 SDK or higher
```
[the remark this answers is not in the document]
Comment (l10nbridge) [reply docs/vs-tools-tizen/how-to-install.md]: Removed

File: docs/vs-tools-tizen/how-to-install.md [hunk 12]
```diff
@@ -1,129 +1,138 @@
...
+To work with Visual Studio Tools for Tizen, your computer must have:
 
-Download the JDK from the [official Oracle Web site](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Select the appropriate platform for your hardware architecture and Windows® version. Then, run the downloaded execution file and follow the displayed instructions.
+- At least 1.5 GB of available disk space
+- Visual Studio 2017  
+Visual Studio Tools for Tizen works with all Visual Studio variations, including Community. Installing or re-installing Visual Studio with .NET desktop development and .NET Core cross-platform development toolsets is recommended.
+- .NET Core 2.0 SDK or higher  
+SDK is needed to set the .NET development environment. For more information, see https://www.microsoft.com/net/download/core.
+- Java Development Kit (JDK) 8 or higher  
+JDK is needed to use the Tizen Baseline SDK. OpenJDK is not supported.
```
[the remark this answers is not in the document]
Comment (l10nbridge) [reply docs/vs-tools-tizen/how-to-install.md]: Added

File: docs/vs-tools-tizen/how-to-install.md [hunk 13]
```diff
@@ -1,129 +1,138 @@
...
+## Visual Studio Tools for Tizen Installation
 
-   * Click the Add button and set URL (Check firewall about ```IP : 10.113.138.168 / Port : 80, 443```)
-      * for Stable Version : http://10.113.138.168/packages/vsext/vsix/content.xml
-      * for Developer Version : http://10.113.138.168/packages/vsext/vsix/content_swc.xml
-   ![Extensions and Gallery](image/HowToInstall_ExtensionsandUpdates2.png)
+To install Visual Studio Tools for Tizen:
 
-   * Click OK button
+1. Set a private gallery for VSIX:
```
[the remark this answers is not in the document]
Comment (l10nbridge) [reply docs/vs-tools-tizen/how-to-install.md]: Removed

File: docs/vs-tools-tizen/how-to-install.md [hunk 14]
```diff
@@ -1,129 +1,138 @@
...
+  - Check the tool path by selecting **Tools &gt; Options &gt; Tizen &gt; Tools**.
+
+    ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
+
+- To check whether Tizen Project properties have been copied correctly:
+  - Make sure that the `Tizen.NET.ProjectType.props` and `Tizen.NET.ProjectType.targets` files are located in the `C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\Tizen\VisualStudio` folder.
+
+     ![Project Property](media/cps-project-property.png)
+
+- To check the Nuget server for build tasks:
```
[the remark this answers is not in the document]
Comment (l10nbridge) [reply docs/vs-tools-tizen/how-to-install.md]: Removed

File: docs/vs-tools-tizen/how-to-install.md (hunk 2 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/how-to-install.md]: Removed

File: docs/vs-tools-tizen/packaging/how-to-create-tpk.md (hunk 9 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/packaging/how-to-create-tpk.md]: Removed

File: docs/vs-tools-tizen/tools/device-manager.md (hunk 5 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/tools/device-manager.md]: Text clarified

File: docs/vs-tools-tizen/tools/device-manager.md (hunk 6 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/tools/device-manager.md]: Text clarified

File: docs/vs-tools-tizen/tools/device-manager.md (hunk 7 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/tools/device-manager.md]: Changed

File: docs/vs-tools-tizen/tools/emulator-manager.md (hunk 1 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/tools/emulator-manager.md]: Link added

File: docs/vs-tools-tizen/tools/log-viewer.md (hunk 8 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/tools/log-viewer.md]: Text clarified

File: docs/vs-tools-tizen/tools/profiler-user-manual.md (hunk 4 above)
Comment (l10nbridge) [reply docs/vs-tools-tizen/tools/profiler-user-manual.md]: Text clarified

**Comment by l10nbridge** [comment]
There are still 2 open queries in files "docs/vs-tools-tizen/packaging/how-to-customize-tpk.md" and "docs/vs-tools-tizen/tools/profiler-user-manual.md"

File: docs/vs-tools-tizen/packaging/how-to-customize-tpk.md (hunk 10 above)
Comment (journey2w) [reply docs/vs-tools-tizen/packaging/how-to-customize-tpk.md]: You had better remove "For more information, see:" section.

Comment (l10nbridge) [reply docs/vs-tools-tizen/packaging/how-to-customize-tpk.md]: Removed

**Comment by journey2w** [comment]
VS Tools team will update soon.

**Pull request merged**